src/Compiler/Driver/CompilerOptions.fs

```diff
@@ -111,7 +111,7 @@ let compilerOptionUsage (CompilerOption (s, tag, spec, _, _)) =
         else
             sprintf "%s:%s" s tag (* still being decided *)
 
-let PrintCompilerOption (CompilerOption (_s, _tag, _spec, _, help) as compilerOption) =
+let PrintCompilerOption print (CompilerOption (_s, _tag, _spec, _, help) as compilerOption) =
```

Comment:
Probably `printer` will be a better name, since we will eventually get `print` in the standard library.

Reply:
Also, what about just having everything return a string and printing it at the end instead of passing the printer function?

Reply:
(Or maybe rather a string list)

Reply:
For now I decided to go for StringBuilder because using sequences requires further refactoring and <s>I'm too lazy for that</s> it's beyond the scope - can be a good followup though.